Terasology/AdventureAssets
# WipeOut is multiplayer ready

The trap is multiplayer ready now. The settings screen sends an event that is handled by the Authority system and the changes are made to the server entity (and further replicated to the client entities).

There is still one problem. The wipeout rod or surfboard cannot be jumped on but can be stepped over gently. This is due to the movement prediction discrepancy. Any entity with a collider behaves weirdly, the player can jerkily pass through it.

### To Test:
- Start Headless + Client 1 + Client 2
- console `give wipeOutRoot`
- Place the block. Change the settings by interacting with the block using `E`
- Check across clients to see if everything works.
